src/middleware.ts

```diff
@@ -16,6 +16,9 @@ export const strictProxyMiddleware = <T extends object>(envObj: T, rawEnv: unkno
     'then',
     // For usage with TypeScript esModuleInterop flag
     '__esModule',
+    // (see #141)
+    'getter',
+    'setter',
```

Comment:
I don't think it's tenable to support framework-specific values here. Instead, how about adding an optional `options` argument to the `strictProxyMiddleware`, which could accept a value like: 

```
{ extraInspectables: ['getter', 'setter'] }
```

Then it would be pretty simple to use a custom `applyMiddleware()` function to solve your issue, without baking fastify-specific details into this library